### DISCUSSION TITLE
Play TTS from HA/MQTT

### DISCUSSION THREAD

**Discussion by masterx1981** [seaky/nspanel_pro_tools_apk#119] [General]
Hi! I've seen that in the future versions of the app, will be possible to play a sound via mqtt. Will ever be possible to use the TTS service?
Thanks!

**Comment by camcs1** [comment]
The HA ADB let's you send TTS commands to it. https://www.home-assistant.io/integrations/androidtv/

Unfortunately the built in TTS voice is quite poor and didn't have much luck loading a 3rd party one

**Comment by masterx1981** [comment]
I was unsuccessful with adb, but notify service, with message "TTS" with data:
ttl: 0
priority: high
media_stream: alarm_stream_max
tts_text: hello

Worked perfectly. 
Thanks!
